hack/quickstart/aws-authenticator-config.yaml

```diff
@@ -0,0 +1,42 @@
+# a unique-per-cluster identifier to prevent replay attacks (see above)
```

Comment:
In the current setup we probably still want bootkube rendering this. Let me open an issue before we address that.